docs/support/coverage.md

```diff
@@ -0,0 +1,27 @@
+# Coverage 
+
+To get the most out of Steamship's cloud platform you'll be using adapters. Steamship adapters serve as drop-in replacement for LangChain's modules. Here's an overview of the status of our adapters and their coverage:
+
+
+## Modules 
+
+* LLMs
+  * ✅ LLM Models
+  * ✅ LLM Caching 
+  * ❌ LLM Serialisation
```

Comment:
i think i asked this elsewhere - what prevents us from supporting serialization?

Reply:
The wrong OpenAI class is being loaded when they load the LLM. A classic namespacing issue in Python. Probably an easy fix for a future PR.

Reply:
Is this related to removing the `_llm_type` property method?

Reply:
Ha, you think so? Going to try and resolve the LLM serialisation tomorrow in an other PR then.